src/mlpack/tests/split_data_test.cpp

```diff
@@ -94,20 +94,35 @@ void CheckDuplication(const Row<size_t>& trainLabels,
     BOOST_REQUIRE_EQUAL(counts[i], 1);
 }
 
-BOOST_AUTO_TEST_CASE(SplitDataResultMat)
+BOOST_AUTO_TEST_CASE(SplitShuffleDataResultMat)
 {
   mat input(2, 10);
-  size_t count = 0; // count for putting unique sequential values
+  size_t count = 0; // counter for unique sequential values
   input.imbue([&count] () { return ++count; });
 
   const auto value = Split(input, 0.2);
-  BOOST_REQUIRE_EQUAL(std::get<0>(value).n_cols, 8); // train data
-  BOOST_REQUIRE_EQUAL(std::get<1>(value).n_cols, 2); // test data
+  BOOST_REQUIRE_EQUAL(std::get<0>(value).n_cols, 8); // Train data
```

Comment:
```suggestion
  BOOST_REQUIRE_EQUAL(std::get<0>(value).n_cols, 8); // Train data.
```

Pedantic style issue.